src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -416,6 +496,53 @@ private function fixNewline(Tokens $tokens, int $index, string $indentation, boo
         $tokens->ensureWhitespaceAtIndex($index + 1, 0, $this->whitespacesConfig->getLineEnding().$indentation);
     }
 
+    private function fixIndentation(Tokens $tokens, int $startIndex, int $endIndex, string $indentation): void
```

Comment:
Indentation is already handled by `statement_indentation` as shown by the integration test in `tests/Fixtures/Integration/priority/method_argument_space,statement_indentation.test`, no need to implement it here.

Maybe we should update this test or add a new one to cover the new option though.